src/games/GameDialogFileList.tsx

```diff
@@ -150,8 +150,8 @@ const GameDialogFileList: React.FC = (props) => {
                 <SortableItem
                   key={x.id}
                   file={x}
-                  onCheckboxChange={(event) => {
-                    setEnabled(x.relative, event.target.checked)
+                  onCheckboxChange={(event, checked) => {
```

Comment:
This isn't necessary for the fix, just a little nicer since the `boolean` is directly passed to the callback